keyring/backends/Windows.py

```diff
@@ -164,7 +164,8 @@ def get_credential(self, service, username):
             res = self._get_password(self._compound_name(username, service))
         # get any first password under the service name
         if not res:
-            res = self._get_password(service)
-            if not res:
-                return None
+            cred = self._get_password(service)
```

Comment:
nitpick: the variable `cred` was used to represent the result from an API call (`_get_password`/`win32cred.CredRead`). Now `cred` is being used for the same purpose. I see why you did it, though. You needed a different name to represent the intermediate result to allow for `res` to represent the value when exiting the branch. Not sure there's anything to be done here.